man/Makefile

```diff
@@ -0,0 +1,13 @@
+Q := @
+
+clean:
+	rm -f elixir.1
+	rm -f iex.1
+
+build:
+	$(Q) cp iex.1.in iex.1
+	$(Q) sed -i"" "/{COMMON}/r common" iex.1
```

Comment:
It won't work with GNU sed=).
The fixed version of Makefile will work with both GNU sed and BSD sed.